src/app/progress/Objective.tsx

```diff
@@ -62,10 +64,18 @@ export default function Objective({
   }
 
   const isBoolean = isBooleanObjective(objectiveDef, completionValue);
+  const showAsCounter =
+    isTrialsPassage &&
+    objectiveDef.allowOvercompletion &&
+    !objectiveDef.allowNegativeValue &&
+    !objectiveDef.allowValueChangeWhenCompleted;
```

Comment:
Isn't this in one of your other helpers?

Reply:
This was a different helper that was only used here but I've put it in `objectives.ts` along with the other helpers.